added support for NI-DAQmx 21.3

RubensRossi opened the pull request:
I added the functions and constants for NI-DAQmx 21.3.0

bjarthur commented:
and thanks for this too!

it appears as if you did not create a separate branch for this PR, and so it includes all the commits for #32 too.  can you address my comments for #32 so we can merge it first?  then i'll take a look at this one.

bjarthur commented:
can you please resolve the conflicts?  would be great to have this too

RubensRossi commented:
conflicts are now solved

bjarthur commented:
thanks!  this looks great.

one question:  i see your comment about VarArgs and Clang v0.13 vs v0.14.  why not use v0.14, or even v0.15?  then those VarArg functions could be called by someone using the low-level API.  so long as everything in NIDAQ.jl works though, i'm fine with this PR as is.  just curious.  seemingly these VarArg functions are new to DAQMx 21.3 as if i search on e.g. `DAQmxSetChanAttribute` src/functions_V20.1.0.jl i don't find it.

RubensRossi commented:
Hi, basically there are just two new functions in the 21.3 if I compare the source API files. The `DAQmxSetChanAttribute` and all the other VarArg are present also in the 20.1. (at first I just copied and renamed the 20.1 function and constant files to 21.3 and everything worked well).
When I added the support for the 21.3, I set the Clang to verbose and then the warnings on VarArg functions popped up. I let them in the file so the issue about the VarArg is visible.
As soon as I have time I will use the latest Clang and make a pull request :)

bjarthur commented:
sorry i took so long to merge!  please bump in future if i don't get to it soon.
